src/com/owncloud/android/ui/fragment/LocalFileListFragment.java

```diff
@@ -98,6 +98,20 @@ public void onActivityCreated(Bundle savedInstanceState) {
         Log_OC.i(TAG, "onActivityCreated() stop");
     }
     
+    public void selectAll(){
```

Comment:
This method should not be added with this feature. Besides, it's not being used.